### PR TITLE
(#1147) Fixed warnings in build process

### PR DESCRIPTION
**Pull request by scrobot**
This is for #1147. We have some problems with fetching some links

```
[WARNING] Error fetching link: https://github.com/yegor256/takes/apidocs/package-list. Ignored it.
[WARNING] Error fetching link: https://github.com/llorllale/cactoos-matchers/apidocs/package-list. Ignored it.
[WARNING] Error fetching link: http://junit.org/apidocs/package-list. Ignored it.
[WARNING] Error fetching link: https://github.com/hamcrest/JavaHamcrest/hamcrest-core/apidocs/package-list. Ignored it.
[WARNING] Error fetching link: https://github.com/hamcrest/JavaHamcrest/hamcrest-library/apidocs/package-list. Ignored it.
[WARNING] Error fetching link: https://github.com/mockito/mockito/apidocs/package-list. Ignored it.
[INFO] 
3 warnings
[WARNING] Javadoc Warnings
[WARNING] javadoc: warning - Error fetching URL: http://docs.mockito.googlecode.com/hg/latest/
[WARNING] javadoc: warning - Error fetching URL: http://www.h2database.com/javadoc/
[WARNING] javadoc: warning - Error fetching URL: http://jolbox.com/bonecp/downloads/site/apidocs/
```

All links are not available, I put some of them directly, other links were disabled from the build. It's the best solution I have. Warnings are disappeared, all java-dock generation not changed.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1168](https://github.com/yegor256/cactoos/issues/1168) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1168](https://github.com/yegor256/cactoos/issues/1168) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/b7a2175b-9a6a-4d26-9df9-f90c804b1769); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#1168` already assigned to @victornoel, can't assign to @victornoel

File: pom.xml [hunk 1]
```diff
@@ -191,7 +191,7 @@ The MIT License (MIT)
                 done. The newly covered classes should be added to the include
                 configuration property below. At the end, the configuration
                 property below should be completely removed so that calls
-                to forbidden APIs always fail the build for every classes.  
+                to forbidden APIs always fail the build for every classes.
```
Comment (victornoel) [inline pom.xml]: @scrobot please do not modify existing todos, it could break things

File: pom.xml [hunk 2]
```diff
@@ -347,7 +358,7 @@ The MIT License (MIT)
           <plugin>
             <groupId>org.codehaus.mojo</groupId>
             <artifactId>sonar-maven-plugin</artifactId>
-            <version>3.4.0.905</version>
+            <version>3.6.0.1398</version>
```
Comment (victornoel) [inline pom.xml]: @scrobot if this is not related to the issue you are solving, please revert this change (if it is, please explain)

File: pom.xml [hunk 3]
```diff
@@ -292,6 +292,17 @@ The MIT License (MIT)
                   <placement>X</placement>
                 </tag>
               </tags>
+              <reportOutputDirectory>${project.basedir}/target</reportOutputDirectory>
+              <destDir>javadoc</destDir>
+              <show>private</show>
+              <detectLinks>false</detectLinks>
+              <detectOfflineLinks>true</detectOfflineLinks>
+              <linksource>true</linksource>
+              <detectJavaApiLink>true</detectJavaApiLink>
```
Comment (victornoel) [inline pom.xml]: @scrobot why do we need all thos new options? Please remove everything that is not needed

File: pom.xml [hunk 4]
```diff
@@ -292,6 +292,17 @@ The MIT License (MIT)
...
+              <links>
+                <link>http://hamcrest.org/JavaHamcrest/javadoc/1.3/</link>
+                <link>https://junit.org/junit4/javadoc/latest/</link>
+              </links>
```
Comment (victornoel) [inline pom.xml]: @scrobot why adding more links? I don't think it is needed, is it?

**Comment by 0crat** [comment]
The architect of the project has changed; @llorllale[/z](https://www.0crat.com/u/llorllale) is not at this role anymore; @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) is the architect now

**Comment by victornoel** [comment]
@scrobot can you take the comments into account?

**Comment by victornoel** [comment]
@paulodamaso I think this PR should be refused, the author does not answer and it does not clearly solve the problem.

**Comment by iakunin** [comment]
@victornoel @paulodamaso I can try to fix this issue instead of the author.

**Comment by victornoel** [comment]
@iakunin let's discuss that in the original ticket, not here, this should be closed

**Comment by paulodamaso** [comment]
@scrobot How about this one? Can I reassign this or you'll provide a solution to it soon?

**Comment by victornoel** [comment]
@paulodamaso can we close this?

**Comment by paulodamaso** [comment]
@victornoel yes, it should be closed, but I don't have this permission right now

**Comment by paulodamaso** [comment]
@yegor256 please, I need permissions in the repository to close pull requests

**Comment by yegor256** [comment]
@paulodamaso granted

**Comment by paulodamaso** [comment]
@yegor256 thanks

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Code review was too long (79 days), architects (@paulodamaso) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
The job [#1168](https://github.com/yegor256/cactoos/issues/1168) is now out of scope

**Comment by 0crat** [comment]
Pull request [#1168](https://github.com/yegor256/cactoos/issues/1168) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by sereshqua** [comment]
@victornoel we would need to make sure that next time all problems found by the code reviewer would be addressed by the pull request author

**Comment by victornoel** [comment]
@sereshqua it would be great yes even though I don't have any power to enforce that :)

**Comment by sereshqua** [comment]
@victornoel I did have to notify you according to policy :)

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)
